src/js/components/DataClearFilters/stories/Simple.js

```diff
@@ -0,0 +1,66 @@
+import React, { useContext } from 'react';
+
+import {
+  Box,
+  Data,
+  DataClearFilters,
+  DataContext,
+  DataFilter,
+  DataFilters,
+  DataSearch,
+  DataSummary,
+  DataTable,
+  SelectMultiple,
+  Toolbar,
+} from 'grommet';
+import { columns, DATA } from '../../DataTable/stories/data';
+
+export const Simple = () => (
+  // Uncomment <Grommet> lines when using outside of storybook
+  // <Grommet theme={...}>
+  <Box pad="medium">
+    <Data
+      data={DATA}
+      total={DATA.length}
+      properties={{
+        location: { label: 'Location' },
+      }}
+    >
+      <DataToolbar />
+      <DataSummary />
+      <DataTable columns={columns} />
+    </Data>
+  </Box>
+  // </Grommet>
+);
+
+const DataToolbar = () => {
+  const { filteredTotal, total } = useContext(DataContext);
+
+  return (
+    <Toolbar align="end">
+      <DataSearch placeholder="Search" />
+      <DataFilters updateOn="change">
+        <DataFilter
+          property="location"
+          // override theme margin to align with search + filter
+          contentProps={{ margin: { bottom: 'none', top: 'xsmall' } }}
+          margin="none"
```

Comment:
```suggestion

```
Is this margin needed?

Reply:
Kind of weird setup but `margin="none"` is for Grommet theme, `contentProps` is for HPE theme. I'll add a comment.

Reply:
oh okay no wonder I didnt see a difference hence my questioning why it was there but thank you for clarifying

Reply:
Done.